### PR TITLE
WEBEU3 - JavaScript-I - Evans Ibok

### PR DESCRIPTION
**Pull request by evansibok**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -75,30 +75,70 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```
Comment (OloruntobiAwoderu) [inline assignments/arrays.js:78]: You got the answer correctly here, but your implementation could be a little better. You could find a way to implement this more programtically

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -5,26 +5,54 @@
...
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+// Stretch Solution
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map((num) => num * 3)
+
```
Comment (OloruntobiAwoderu) [inline assignments/function-conversion.js:57]: Great research and use of map

File: assignments/objects.js [hunk 3]
```diff
@@ -49,16 +64,33 @@ const example = {
...
+  age: 70,
+  child: {
+    name: 'George',
+    age: 50,
+    grandchild: {
+      name: 'Sam',
+      age: 30,
+    }
+  }
+}
```
Comment (OloruntobiAwoderu) [inline assignments/objects.js:78]: Good work

**Review by OloruntobiAwoderu — COMMENTED** [review]
Great work overall on this project Evans!!!!
You completed MVP and all the stretch in spite of all your difficulties
I'm really impressed
Keep up the good work

File: assignments/arrays.js (hunk 1 above)
Comment (evansibok) [reply assignments/arrays.js:78]: I tried to get the ID of the 33rd car in the array but this was the only way I could find. If there's another way I'll appreciate if you can show me. Thank you!

**Comment by evansibok** [comment]
> Great work overall on this project Evans!!!!
> You completed MVP and all the stretch in spite of all your difficulties
> I'm really impressed
> Keep up the good work

Thank you! To be honest it wasn't easy. I had to stay up till 1am to complete this because we had light but I'm glad I did.

**Pull request merged**